test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -29,6 +29,20 @@ def __str__(self):
         return self.__repr__()
 
 
+class OneOf(object):
```

Comment:
This is definitely a hack, but because of inconsistent `os.path.realpath` type calls, dbt doesn't really support the synlink shenanigans I get up to in the test suite. As a result, a number of paths can be either the symlinked path or the real path, and that changes during compilation when fields get manipulated. I didn't want this project to turn into changing how dbt calculates paths, so I did this instead.